PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/utility/AzureAnAction.java

```diff
@@ -69,7 +90,15 @@ public void sendTelemetryOnAction(AnActionEvent anActionEvent, final String acti
         if (this instanceof TelemetryProperties) {
             properties.putAll(((TelemetryProperties) this).toProperties());
         }
-        AppInsightsClient.createByType(AppInsightsClient.EventType.Action, anActionEvent.getPresentation().getText(), null, properties);
+        return properties;
+    }
+
+    protected String getServiceName() {
+        return "";
```

Comment:
Just return "ACTION" here and no need to check whether the return value is Empty in actionPerformed